chain/types/ethtypes/eth_transactions.go

```diff
@@ -113,10 +113,6 @@ func EthTransactionFromSignedFilecoinMessage(smsg *types.SignedMessage) (EthTran
 	if smsg == nil {
 		return nil, errors.New("signed message is nil")
 	}
-	// Validate the sender's address format.
```

Comment:
Mpool validation/Block validation already enforces that the sender should either be an f4 address or an ID address that can be resolved to an f4 address. No harm in allowing legacy txns to have ID addresses as sender.

Reply:
Consider making this explicit in the FIP, I don't think it currently is.

Reply:
Filed at https://github.com/filecoin-project/FIPs/pull/1019.